### PR TITLE
add driver and simplecache example

### PR DESCRIPTION
**Pull request by aaronspring**

### CODE REVIEW

File: docs/source/tutorial.ipynb [hunk 1]
```diff
@@ -2008,13 +2008,132 @@
     "ds"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Working with nested catalogs\n",
```
Comment (aaronspring) [inline docs/source/tutorial.ipynb:2015]:
```suggestion
    "## Caching\n",
```

**Comment by aaronspring** [comment]
I should have directly commited into #19. instead this PR want to commit into NCAR:add-docs

I resolved conflicts, https://intake-thredds--20.org.readthedocs.build/en/20/tutorial.html#id1 looks good. But I am unsure how to merge this PR into master. or should I just open a new PR?

**Comment by andersy005** [comment]
I changed the base branch to `main` via the `edit` option

Screen Shot 2021-02-17 at 12 36 21 PM

Can you try rebasing against the upstream/main to address the conflicts? Sorry for the extra work :)

File: docs/source/tutorial.ipynb [hunk 2]
```diff
@@ -2009,6 +2009,132 @@
...
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import shutil\n",
+    "\n",
+    "shutil.rmtree('my_caching_folder')"
+   ]
+=======
```
Comment (andersy005) [inline docs/source/tutorial.ipynb:2137]:
```suggestion
```

File: docs/source/tutorial.ipynb [hunk 3]
```diff
@@ -2009,6 +2009,131 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import shutil\n",
+    "\n",
+    "shutil.rmtree('my_caching_folder')"
+   ]
```
Comment (andersy005) [inline docs/source/tutorial.ipynb:2136]: Not 100% sure, but I think there's a missing comma

```suggestion
   ],
```

**Comment by aaronspring** [comment]
image

**Comment by andersy005** [comment]
@aaronspring, yeah... I saw that. It's weird that the cell fails but everything else after it still works. Also, I notice some weird dask dependency issues in #19. Let's merge this as is for now and investigate the issue in a separate PR if need be.

**Pull request merged**